modules/images/webp-uploads/load.php

```diff
@@ -742,3 +742,33 @@ function webp_uploads_get_image_sizes_additional_mime_type_support() {
 
 	return $allowed_sizes;
 }
+
+/**
+ * Updates the quality of images used in the wp editor.
+ * For WP version less than 6.2 returns 82,
+ * Otherwise returns default quality.
+ *
+ * @since n.e.x.t
+ *
```

Comment:
```suggestion
 *
 * @global string $wp_version The WordPress version string.
 *
```

Add missing doc for `global`